core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalLogWriter.java

```diff
@@ -238,13 +238,17 @@ public synchronized void flush() throws IOException {
   }
 
   @Override
-  public synchronized void close() throws IOException {
-    Closer closer = Closer.create();
-    if (mJournalOutputStream != null) {
-      closer.register(mJournalOutputStream);
+  public synchronized void close() {
+    try {
+      Closer closer = Closer.create();
+      if (mJournalOutputStream != null) {
+        closer.register(mJournalOutputStream);
+      }
+      closer.register(mGarbageCollector);
+      closer.close();
+    } catch (IOException e) {
+      LOG.warn("Failed to close UfsJournalLogWriter with error: {}.", e.getMessage());
```

Comment:
This was necessary for hadoop1. I've removed this. 

I think it was fine to swallow this because this is only called when a leader master is stopped. Leader master is usually stopped with an unexpected failure.